api-reference/10 UI Components/dxTabPanel/dxTabPanel.md

```diff
@@ -6,7 +6,7 @@ inherits: dxMultiView
 ---
 ---
 ##### shortDescription
-The TabPanel is a UI component consisting of the [Tabs](/concepts/05%20UI%20Components/Tabs/00%20Overview.md '/Documentation/Guide/UI_Components/Tabs/Overview/') and [MultiView](/concepts/05%20UI%20Components/MultiView/00%20Overview.md '/Documentation/Guide/UI_Components/MultiView/Overview/') UI components. It automatically synchronizes the selected tab with the currently displayed view and vice versa.
+The TabPanel is a UI component consisting of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/concepts/05%20UI%20Components/MultiView/00%20Overview.md '/Documentation/Guide/UI_Components/MultiView/Overview/') UI components. It automatically synchronizes the selected tab with the currently displayed view and vice versa.
 
```

Comment:
I have a question regarding line 23 below.

Line 23:
text: "CA San Francisco Stanford Ave st."

Is "st" a typo?

Reply:
Yes, I guess it's a typo, I will change it.

Reply:
Thank you.